server/tests-py/test_graphql_mutations.py

```diff
@@ -184,6 +184,18 @@ def dir(cls):
         return "queries/graphql_mutation/insert/constraints"
 
 
+@pytest.mark.parametrize("transport", ['http', 'websocket'])
+@usefixtures('per_class_tests_db_state')
+class TestGraphqlInsertNullPrefixedColumnOnConflict:
```

Comment:
I think this could probably have been added to one of the existing insert classes, since we can setup multiple unrelated tables for different tests in one class setup file. I don't think we need to change it now though.